### DISCUSSION TITLE
How do I split embroidery files? Please help!

### DISCUSSION THREAD

**Discussion by ghost** [inkstitch/inkstitch#1341] [Q&A]
I already have PES/SVG files but I have a PE550D machine that only goes up to a 4x4 hoop, and my files I want to embroider are all bigger than that… I really need help splitting embroidery files :(

**Comment by tyrosinase** [comment]
The bad news is... that's almost impossible to do and get aligned correctly. The PE770/PE800 has a (third-party) hoop that'll do 5x12, using multiple attachment points to get overlapping 5x7 fields without re-hooping, and even _that_ is imperfect - the most stable fabric/stabilizer setup is still going to shift as the stitches pull.

If you're just breaking up pieces that don't have to align perfectly though, you can maybe do it. I wrote a whole tutorial on using the 5x12 hoop, and most of what I talk about still applies: https://silverseams.com/tutorials/using-a-5x12-hoop-on-a-5x7-embroidery-machine/

**Reply by ghost** [reply]
Oh :( So Inkstitch is probably not a good idea… If I save enough money for like Embrilliance software where it CAN split a design - will that be better for me?

**Reply by tyrosinase** [reply]
Maybe. But probably not.

The problem is mechanical, not software: if you stitch one half of a fill, you have to hoop it in a way that it will *exactly* line up, stitch for stitch. And that just doesn't happen, even in that 5x12 hoop where you don't have to try to re-hoop precisely.

Take a look at Urban Threads' split designs, to get an idea of what it's realistically possible to split: https://urbanthreads.com/search.aspx?qs=split&df=Machine If you have designs that have natural break points, then you can split just fine with Ink/Stitch. If you don't, there's nothing Embrilliance can really do either.

Basically, you can do it with designs that overlap, but not things that align precisely. If you look through those UT designs, many of them break between elements that don't touch, but many of them break "between" overlapping elements that are casually arranged, so it won't be obvious if it's shifted up to a few millimeters in any direction.
